src/app/client/src/app/modules/core/components/main-header/main-header.component.html

```diff
@@ -60,6 +60,10 @@
                           <a class="item" *ngIf='permissionService.permissionAvailable' appPermission [permission]='orgAdminRole' routerLink="/org/userUpload">
                             {{resourceService.frmelmnts?.lbl?.manageuser}}
                           </a>
+                          <a class="item" *ngIf='permissionService.permissionAvailable' appPermission
+                              [permission]='adminDashboard' routerLink="manage">
```

Comment:
"adminDashboard" : ["ORG_ADMIN","SYSTEM_ADMINISTRATION","REPORT_VIEWER"],
this is the roles for which the manage link will be visibel
but I hope this is required for only ORG_ADMIN and SYSTEM_ADMINISTRATION role please use 
"orgAdminRole" : ["ORG_ADMIN","SYSTEM_ADMINISTRATION"]

Reply:
Done, Thanks for clarification.